telephony/android_modem.h

```diff
@@ -239,6 +239,11 @@ extern void amodem_set_gsm_location( AModem modem, int lac, int ci );
 extern int amodem_get_base_port( AModem  modem );
 extern int amodem_get_instance_id( AModem  modem );
 
+/** Enable/Disable the selected modem feature
+ **/
+
+extern int amodem_enable_disable( AModem  modem, const char*  feature, bool  enable );
```

Comment:
Hi vicamo,

Thanks for your advice. I think having a enum is a great idea for a better extensibility, but I don't understand how we can benefit from defining another function ```do_gsm_enable_disable(...)``` rather than calling the the ```amodem_enable_feature(...)``` directly.